perun-registrar-lib/src/test/java/cz/metacentrum/perun/registrar/ExpirationNotifSchedulerTest.java

```diff
@@ -686,4 +751,81 @@ private AttributeDefinition setUpGroupMembershipExpirationAttribute() throws Exc
 
 		return perun.getAttributesManager().createAttribute(session, attr);
 	}
+
+	/**
+	 * In test DB in application_form is created unique index for vo_id without dependency on group_id, so if you
+	 * want to create application form for group, firstly you have to delete automatically created form for Vo.
+	 * Then method creates new group.
+	 *
+	 * @return created group
+	 * @throws GroupExistsException
+	 */
+	private Group deleteApplicationFormAndCreateGroup() throws GroupExistsException {
+		// delete automatically created form for Vo
+		jdbc.update("DELETE from application_form WHERE vo_id = ?", vo.getId());
```

Comment:
I forgot to mention, that you should call this only for HSQLDB, since it doesn't apply to the Postgres DB. See ``Compatibility.isHSQLDB()``.

Reply:
It's true! So edited